### ISSUE TITLE
Documentation incorrect

### ISSUE THREAD

**Issue by 31337-4554551n** [DarkFlippers/unleashed-firmware#205]
### Describe the bug.

Documentation says to put the file in the "playlist" folder
The folder it looks for is "subplaylist"

### Reproduction

create a playlist folder
Add a playlist
open playlist app
See it defaulting to a different directory
Move file to subplaylist folder
open app
see it in the list

**Comment by UberGuidoZ** [comment]
Could you link to the documentation? There are different utilities "playlist" and "subplaylist" depending on the application and firmware. Thank you.

EDIT: [Is this the location you are referencing](https://github.com/DarkFlippers/unleashed-firmware/pull/62)? If so, this is a pull request and not the official documentation. It looks like the documentation needs to be made and placed into the [Documentation directory](https://github.com/DarkFlippers/unleashed-firmware/tree/dev/documentation).

Please feel free to create proper documentation and submit a PR! We always appreciate the help. =)

**Comment by 31337-4554551n** [comment]
Yes, this was the one. Guess I can't tell the difference between what's official and what isn't yet. I followed the link from your front page where it talks about the play list plugin by clicking on the link next to it. 
![Screenshot_20221220_123627_Firefox](https://user-images.githubusercontent.com/26422914/208568754-077a1b0b-9f4d-4d37-bdf5-8edf0f8b418f.jpg)

**Comment by xMasterX** [comment]
This is link to original source, which is also outdated

Documentation for playlist plugin does not exist in our repo, will be done later

**Comment by mishamyte** [comment]
Looks like stale, if still a point - pls open a fresh ticket. Thanks
